resources/building-mobile-extensions/event-processing.md

```diff
@@ -1,42 +1,44 @@
 # Event processing
 
-An event is composed of the following information:
+An event is composed of the following components:
 
-* A name, which is used for debugging.
-* An event type, which defines what kind of event this is and determines who is notified when this event occurs.
-* An event source, which is where the event originated and is used to determine who is notified when this event occurs.
-* Event data, which is the structured data that is associated with this particular event. This data is retrievable as a JSON blob.
-* Versioning information, which is used to synchronize a group of events with the associated shared data, is not directly exposed to extensions. This information is implicitly used when getting shared states.
+* A name, which is used for debugging
+* An event type, which defines what kind of event this is and determines who is notified when this event occurs
+* An event source, which is where the event originated and is used to determine who is notified when this event occurs
+* Event data, which is the structured data that is associated with this particular event. This data is retrievable as a JSON blob
+* Versioning information, which is used to synchronize a group of events with the associated shared data, is not directly exposed to extensions. This information is implicitly used when getting shared states
```

Comment:
Why are we removing the sentence periods on this section above?

Reply:
Generally for lists we don't need periods - but this one is just more of a me preference than an Adobe-styling preference, so if you'd like me to re-add the periods, I can add them in!

Reply:
Also, generally, because bulleted lists aren't sentences (especially in this case and in the cases below), so I don't think it makes sense to put periods at the end.